### ISSUE TITLE
plot3D with plotly

### ISSUE THREAD

**Issue by rodemann** [jakobbossek/smoof#131]
Hi, 
when plotting two-dimensional single-objective functions with plotly, the axes are not labelled correctly. E.g. for PriceN4Function the global optima in the plot are somewhere near (50,50), while they actually are around (0,0). Same holds for Branin and other 2D Functions with one target only. Thanks for your help in advance!

```
require(plotly)
require(smoof)
plot3D(makePriceN4Function(), package = "plotly")
plot3D(makeBraninFunction(), package = "plotly")
```

**Comment by jakobbossek** [comment]
Will have a look at this. Thanks for the pointer!

**Comment by jakobbossek** [comment]
This was a general bug (see [this post](https://stackoverflow.com/questions/39576455/plotly-3d-surface-graph-has-incorrect-x-and-y-axis-values) for details).
Fixed it in e9b18a3.

By the way: sorry for the way too late answer!
